packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -271,9 +274,12 @@ function useFocusContainment(scopeRef: RefObject<HTMLElement[]>, contain: boolea
 
   // eslint-disable-next-line arrow-body-style
   useEffect(() => {
-    let rafRef = raf.current;
-    return () => cancelAnimationFrame(rafRef);
-  }, []);
```

Comment:
Hrm, I'm not sure about this logic here. Wouldn't this store the initial `raf` value (i.e. its initialized null value) and never update it since `raf` isn't in the dep array?

Reply:
yep, that's accurate, i was perhaps a bit overzealous with changing some of those